data/json/items/tool_armor.json

```diff
@@ -2633,5 +2633,51 @@
     "revert_to": "foodperson_mask",
     "use_action": "FOODPERSON",
     "flags": [ "OUTER", "SUN_GLASSES", "NO_TAKEOFF", "LIGHT_10", "WATCH", "NO_UNLOAD" ]
+  },
+  {
+    "id": "attached_ear_plugs_on",
+    "type": "ARMOR",
+    "name": "pair of attached ear plugs (in)",
+    "name_plural": "pairs of attached ear plugs",
+    "description": "A pair of Industrial grade ear plugs, they're attached together by some string. they're inside your ears, use them to take them out.",
```

Comment:
```suggestion
    "description": "A pair of Industrial grade ear plugs attached together by some string. They're inside your ears, activate them to take them out.",
```

Reply:
You need the majuscule, but for the rest of the suggestion I'm not sure. I think activate is clearer because that's what the UI tells you and removing `, they're` streamline the sentence a bit.

Reply:
```suggestion
    "description": "A pair of industrial grade ear plugs attached together by some string.  They're inside your ears, activate them to take them out.",
``` 

Double-spacing.